src/locales/en/common.json

```diff
@@ -2747,6 +2747,11 @@
       "flash-bootloader": "Bootloader updating",
       "flash": "Flashing your device",
       "firmware": "Firmware updating"
+    },
+    "newVersion": "Update Firmware to {{version}} is available",
+    "drawerUpdate": {
+      "title": "Firmaware Update",
+      "description": "Update your Ledger Nano firmware by connecting it to Ledger Live application on desktop"
```

Comment:
Just a small gramatical thing: could you change it to `"Update your Ledger Nano firmware by connecting it to _the_ Ledger Live application on desktop"` @klambert-ledger

Reply:
Done 👍